### DISCUSSION TITLE
OtpInput with integerOnly, tab and enter do not work in forms.

### DISCUSSION THREAD

**Discussion by ManoLuuL** [primefaces/community#1845] [PrimeReact] [answered]
Using the OtpInput component, the following cases are being observed.

1° When all fields are filled with a value and then attempting to use the Tab key to move between other components or even between the values, the action does not occur. Another case is when there is a form and we try to use Enter for submission, but the action also does not occur. However, if we leave the value incomplete in the component, both Tab and Enter work.
Example on the Prime website: https://primereact.org/inputotp/#basic

2° When using the integerOnly prop, tab navigation does not work from the beginning of the component, thus also not functioning with Enter if in a form.
Example on the Prime website: https://primereact.org/inputotp/#integeronly

**Comment by nitrogenous** [comment, marked as answer]
Thank you for bringing this to our attention. It appears that this issue occurred during the accessibility implementation. We have noted your bug report and successfully reproduced the issue. A task will be created to address this, and we will work to resolve it as soon as possible.

Issue Link: https://github.com/primefaces/primereact/issues/6632
